cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -472,11 +472,14 @@ def ANSIBLE_PLAYBOOK(self):  # noqa
     def instantiate(self, catalog_item_class, *args, **kwargs):
         return catalog_item_class.from_collection(self, *args, **kwargs)
 
-    def create(self, catalog_item_class, *args, **kwargs):
+    def create(self, catalog_item_class, provider_type=None, field_entry_point=False, *args,
```

Comment:
Are you able to access `provider` `attribute` in a method?
if yes then 
Required: you can use internally `provider.category`. don't pass it form test to `create` method

Reply:
@digitronik 
I am not able access **provider** attribute directly in ```create``` method but  I can pass provider attribute to create method from test like instead of passing ```provider=provider.category```; we can pass this - ```provider=provider```
Please let me know if you have any other solution around it.
Thank you.